components/os/component_output.go

```diff
@@ -362,31 +363,35 @@ type MachineMetadata struct {
 var currentMachineMetadata MachineMetadata
 
 func init() {
-	// e.g., "/proc/sys/fs/file-max" exists on linux
-	if file.CheckFDLimitSupported() {
-		limit, err := file.GetLimit()
-		if limit > 0 && err == nil {
-			// set to 20% of system limit
-			DefaultZombieProcessCountThreshold = int(float64(limit) * 0.20)
+	// Linux-specific operations
+	if runtime.GOOS == "linux" {
```

Comment:
if runtime.GOOS != linux (typedef this) {
   return
}

Reply:
done